### DISCUSSION TITLE
can I open url and autotype in one operation without the browser addon ?

### DISCUSSION THREAD

**Discussion by bobOnGitHub** [keepassxreboot/keepassxc#8578] [Q&A] [answered]
Just trying out keepassxc 2.7.1 on Alma Linux 8.6 , KDE 5.23

Ctrl-shift-u works : open the web page in my browser (Firefox)
Ctrl-shift-v works : auto type the specified sequence on the web page. (asks if ok to type in prev active window)

I can't find any settings/doc to allow me to do all this in one go - I'm looking for a way to do it without using the browser addon.

As it is I have to click back on keepassxc after the web page opens (even with keepassxc as "always on top") and then autotype.

Can it be done without the browser addon and if so how ?

**Comment by droidmonkey** [comment, marked as answer]
Use the global Auto-Type shortcut. We can't "do it in one go" because we have no idea when the webpage has finished loaded, whether the login fields have focus, or you are even on a page that is for logging in at all. Read the user guide for more information.

**Reply by bobOnGitHub** [reply]
Cheers @ droidmonkey,

I thought there might be a way of setting up an "open,wait,type" sequence to allow page to load / user to click on first field if necessary
Eg. autotype seq: {OPEN-URL}{WAIT 750}{USERNAME}{TAB}{PASSWORD}
I leave the final {ENTER} out so I can see all ok before hitting return...

Global autotype with title matching is probably better in most cases as no wait reqd and not switching to keepassxc window first.
(I gather url matching requires the browser extension)

thanks again.

**Reply by droidmonkey** [reply]
Url matching only works if the window title has the url in it.
